packages/admin/admin/src/table/filterbar/FilterBar.tsx

```diff
@@ -1,46 +1,64 @@
-import { ComponentsOverrides, Theme } from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides, css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
+import { Root as FieldContainerRoot } from "../../form/FieldContainer";
```

Comment:
@jamesricky isn't it possible to achieve the same thing by using `FieldContainer`? It's strange that the FilterBar needs to know about the elements in FieldContainer...

Reply:
I don't think it's possible, as we need to target a html-element in the selector, not a component with multiple elements... I tried forwarding the `ref` from `FieldContainer` to it's `Root` slot but that didn't change anything 🤷🏼‍♂️